challenges/desafio24.js

```diff
@@ -0,0 +1,15 @@
+db.voos.find(
+  {
+    $and: [
+      {litrosCombustivel: {$not: {$gt: 600}}},
+      {"empresa.nome": {$nin: ["GOL", "AZUL"]}},
+      {litrosCombustivel: {$exists: true}}
+    ]
+  },
+  {
+    vooId: 1,
+    litrosCombustivel: 1,
+    "empresa.nome": 1,
+    _id: 0
+  }
+).limit(1).pretty();
```

Comment:
Outra maneira com o $and implícito e sem o limit(1):
```suggestion
db.voos.findOne({ litrosCombustivel: { $lte: 600, $exists: true }, "empresa.nome": { $nin: ["GOL", "AZUL"] } }, { vooId: 1, "empresa.nome": 1, litrosCombustivel: 1, _id: 0 });
```
Ai é aquele role que eu falei sobre o $and implicito, a gente nao pode repetir chaves, entao a query referente a "litrosCombustivel" tem que estar toda em uma chave.... se eu coloco "litrosCombustivel" duas vezes (umpa pro limite e outra pro exists) daria ruim!